src/AppBundle/Form/UserRegitType.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nima
+ * Date: 25.12.16
+ * Time: 14:35.
+ */
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Security\Core\User\User;
+
+class UserRegitType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            //->add('name')
+            //->add('save', SubmitType::class)
+
+            ->add('login', TextType::class, [
+                'required' => false,
+                'label' => 'login',
+                'attr' => ['class' => 'test col-xs-3'],
+            ])
+
+            ->add('firstName', TextType::class, [
+                'required' => false,
+                'label' => 'firstName',
+                'attr' => ['class' => 'test col-xs-6'],
+            ])
+            ->add('lastName', TextType::class, [
+                'required' => false,
+                'label' => 'lastName',
+                'attr' => ['class' => 'test col-xs-6'],
+            ])
+                      ->add('email', TextType::class, [
+                    'required' => false,
+                    'label' => 'email',
+                    'attr' => ['class' => 'test col-xs-3'],
+              ])
+
+          ->add('city', TextType::class, [
+           'required' => false,
+           'label' => 'city',
+           'attr' => ['class' => 'test col-xs-6'],
+           ])
+         ->add('address', TextType::class, [
+        'required' => false,
+        'label' => 'address',
+        'attr' => ['class' => 'test col-xs-6'],
+    ])
+
+          ->add('password', TextType::class, [
```

Comment:
i'm almost sure that it should be a PasswordType and plus desirable RepeatedType